eng/scripts/Copy-ApiVersion.ps1

```diff
@@ -38,18 +38,25 @@ param (
     [Parameter(Mandatory = $true)]
     [ArgumentCompleter({ "preview/", "stable/" })]
     [ValidateScript({ 
-            function script:Get-Version([string] $version) {
-                if ($version -match '^(?<status>stable|preview)/(?<version>(\d{4}-\d{2}-\d{2}|\d+\.\d+)(-preview(\.\d+)?)?)$') {
-                    return $Matches['status'], $Matches['version']
-                }
-        
-                throw 'Version must start with "stable/" or "preview/" and end with either a date-based version (recommended) like 2024-01-05 or a major.minor semver, followed by an optional "-preview" for previews APIs.'
+        function script:Get-Version($version) {
```

Comment:
Can you get rid of the function declaration here too given you're inlining the same logic in the second validation script?

Reply:
It's used in the body as well. This used to work having just once here, but PowerShell seems to have broken that. I don't want to define it thrice, and validation scripts can't use functions defined in the body or not yet imported.